roles/remediate/tasks/leapp_cgroups-v1_enabled.yml

```
@@ -0,0 +1,68 @@
+---
+- name: leapp_cgroups-v1_enabled | Disable cgroups-v1 in favor of the new version cgroups-v2
+  vars:
+    leapp_report_location: /var/log/leapp/leapp-report.json
+    entry_title: cgroups-v1 enabled on the system
+  when: ansible_distribution == 'RedHat' and ansible_distribution_major_version|int == 9
+  block:
+    - name: leapp_cgroups-v1_enabled | Check that the leapp-report.json exists
+      ansible.builtin.stat:
+        path: "{{ leapp_report_location }}"
+      register: leapp_report_stat
+
+    - name: leapp_cgroups-v1_enabled | End remediation early if no leapp report exists
+      ansible.builtin.set_fact:
+        leapp_report_missing: true
+      when: not leapp_report_stat.stat.exists
+      failed_when: not leapp_report_stat.stat.exists
+
+    - name: leapp_legacy_network_config_migration | Read leapp report
+      ansible.builtin.slurp:
+        src: "{{ leapp_report_location }}"
+      register: leappreport
+
+    - name: leapp_cgroups-v1_enabled | Parse leapp report
+      ansible.builtin.set_fact:
+        leappreportdata: "{{ leappreport.content | b64decode | from_json }}"
+
+    - name: leapp_cgroups-v1_enabled | Find the inhibitor entry and extract the remediation command
+      # The remediation executes recommended command from the report since it
+      # is generated with specific kernel arguments that need to be disabled
+      # based on the system's configuration.
+      ansible.builtin.set_fact:
+        remediation: "{{ item.detail.remediations | selectattr('type', 'eq', 'command') | first }}"
+      loop: "{{ leappreportdata.entries }}"
+      when: item.title == entry_title and (item.detail.remediations | selectattr('type', 'eq', 'command') | list | length > 0)
+
+    - name: leapp_cgroups-v1_enabled | End remediation if no matching entry was found in the leapp report
+      ansible.builtin.set_fact:
+        leapp_report_missing: true
+      failed_when: remediation is not defined
+
+    - name: leapp_cgroups-v1_enabled | Output the remediation command
+      ansible.builtin.debug:
+        msg: "{{ remediation.context | join(' ') }}"
+
+    - name: leapp_cgroups-v1_enabled | Update the kernel arguments to disable the cgroups-v1 in favor of cgroups-v2
+      ansible.builtin.command: "{{ remediation.context | join(' ') }}"
+      when: remediation is defined
+      register: leapp_disable_cgroupsv1
+      changed_when: leapp_disable_cgroupsv1 is success
+
+    - name: leapp_cgroups-v1_enabled | Output the remediation command result
+      ansible.builtin.debug:
+        msg: "{{ leapp_disable_cgroupsv1.stdout }}"
```

```suggestion
        msg: "{{ leapp_disable_cgroupsv1.stdout_lines }}"
```

If the output is large, consider using stdout_lines for prettier printing. I do not know what's the output so decide yourself.